x/mint/params.go

```diff
@@ -70,3 +86,15 @@ func (p Params) String() string {
 		p.InflationMin, p.GoalBonded, p.BlocksPerYear,
 	)
 }
+
+// Implements params.ParamSet
+func (p *Params) ParamSetPairs() params.ParamSetPairs {
+	return params.ParamSetPairs{
```

Comment:
it'd be cool to use `params.NewParamSetPairs` here

Reply:
I don't think we need `New*` constructor for slice literals, but certainly their contents.